images/k8s-rdma-shared-dev-plugin-ds-pre-1.16.yaml

```diff
@@ -15,6 +15,9 @@ spec:
         name: rdma-shared-dp-ds
     spec:
       hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+        feature.node.kubernetes.io/custom-rdma.capable: "true"
```

Comment:
> feature.node.kubernetes.io/custom-rdma.capable: "true"
 this depends on NFD v0.6.0 (or manually marking the node with this label which i dont think we want)

i am not too sure we want to add this dependency here , but rather just add a section in README about how the nodes should be selected. (with a nodeSelector snippet)